parts/k8s/kubernetesagentresourcesvmas.t

```diff
@@ -46,8 +46,10 @@
           {{end}}
         ]
 {{if not IsAzureCNI}}
+  {{if not IsAzureStackCloud}}
         ,
         "enableIPForwarding": true
```

Comment:
@jackfrancis  AzureStack enable the IP forwarding by default and can not change the setting. So I add the logic to omit the filed for Azure Stack.